src/main/java/piglatin/PigLatinConverter.java

```diff
@@ -13,7 +13,17 @@ public static boolean isVowel(char letter) {
 	
 	public static String getInputFromCommand() {
 		Scanner scan = new Scanner(System.in);
-		String input = scan.next();
+		String input = "";
+		
+		try {
+			input = scan.nextLine();
+		}
+		catch (java.util.NoSuchElementException e) {
+			System.out.println("NoSuchElementException occured while scanning input!");
+		}
+		catch (java.lang.IllegalStateException e) {
+			System.out.println("IllegalStateException occured while scanning input!");
+		}
```

Comment:
Please format the above as

try {

} catch (ExceptionType name) {

} catch (ExceptionType name) {

}